notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb

```diff
@@ -411,6 +358,21 @@
     "Note that MLFlow model registration is strictly only required if accessing the trained model at a later time. For training the model, and performing inferencing in the same notebook, the model object model is sufficient."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if running_on_synapse():\n",
```

Comment:
Nit: Please add a prerequisites cell in the top of this notebook and add a parameter for the workspace name so that people will 1) Know how to create and set up the linked service. 2) Know that they need to change that

Reply:
Thanks! AML workspace name is set in linked service. I created a doc about how to create AML and linked service (in hyperOpt PR) and put a link in prerequisite. The doc will be available after that one merged in.